### PR TITLE
Feature/77649

### PR DESCRIPTION
**Pull request by JKrupinski**
[#77649](https://dev.snowdog.pro/issues/77649)

### CODE REVIEW

File: PayPal_Braintree/styles/modules/_apple-pay.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.braintree-apple-pay-button {
+    -webkit-appearance: -apple-pay-button;
+    display: block;
+    padding: $spacer 0;
+    margin: $spacer--medium 0;
+    width: 200px;
+    min-height: 40px;
```
Comment (anqaka) [inline PayPal_Braintree/styles/modules/_apple-pay.scss:7]: shouldn't be 48px for the button as a minimal height?

File: PayPal_Braintree/styles/modules/_google-pay.scss [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+    background-repeat: no-repeat;
+    background-size: contain;
+    border: 0;
+    border-radius: $spacer--extra-small;
+    box-shadow: rgba(60, 64, 67, 0.3) 0 1px 1px 0, rgba(60, 64, 67, 0.15) 0 1px 3px 1px;
+    cursor: pointer;
+    height: 40px;
+    min-height: 40px;
+    padding: $spacer--small $spacer--semi-medium;
+}
```
Comment (anqaka) [inline PayPal_Braintree/styles/modules/_google-pay.scss:13]: I'd create variables for this file, might be in the same file but to be easy to change some value, as:
`box-shadow`, `border-radius`,  `height` & `paddings`

File: PayPal_Braintree/styles/modules/_google-pay.scss [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+button.braintree-googlepay-button.black {
+    background-color: #000;
```
Comment (anqaka) [inline PayPal_Braintree/styles/modules/_google-pay.scss:16]: we have variable `$black` for this one :)

File: PayPal_Braintree/styles/modules/_google-pay.scss [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+    padding: $spacer--small $spacer--semi-medium $spacer;
+}
+
+button.braintree-googlepay-button.white {
+    background-color: $white;
+}
+
+button.braintree-googlepay-button.short {
+    min-width: 90px;
+    width: 160px;
```
Comment (anqaka) [inline PayPal_Braintree/styles/modules/_google-pay.scss:27]: to variables

File: PayPal_Braintree/styles/modules/_google-pay.scss [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+}
+
+button.braintree-googlepay-button.black.short {
+    background-image: url(https://www.gstatic.com/instantbuy/svg/dark_gpay.svg);
```
Comment (anqaka) [inline PayPal_Braintree/styles/modules/_google-pay.scss:31]: to variable

File: PayPal_Braintree/styles/modules/_google-pay.scss [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+}
+
+button.braintree-googlepay-button.white.short {
+    background-image: url(https://www.gstatic.com/instantbuy/svg/light_gpay.svg);
```
Comment (anqaka) [inline PayPal_Braintree/styles/modules/_google-pay.scss:35]: to variable

File: PayPal_Braintree/styles/modules/_google-pay.scss [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+    width: 240px;
+}
+
+button.braintree-googlepay-button.white.long {
+    background-image: url(https://www.gstatic.com/instantbuy/svg/light/en.svg);
+}
+
+button.braintree-googlepay-button.black.long {
+    background-image: url(https://www.gstatic.com/instantbuy/svg/dark/en.svg);
+}
```
Comment (anqaka) [inline PayPal_Braintree/styles/modules/_google-pay.scss:69]: above - images, colors codes, heights -> to variables please :)

**Review by anqaka — CHANGES_REQUESTED** [review]

File: PayPal_Braintree/styles/modules/_apple-pay.scss (hunk 1 above)
Comment (JKrupinski) [reply PayPal_Braintree/styles/modules/_apple-pay.scss:7]: hmm it can be 48px :)

File: PayPal_Braintree/styles/modules/_google-pay.scss (hunk 7 above)
Comment (anqaka) [reply PayPal_Braintree/styles/modules/_google-pay.scss:136]: a lot of variables, I was thinking about a few of them but it looks great!

**Review by anqaka — APPROVED** [review]

**Pull request merged**